control/veda_deliver.py

```diff
@@ -629,6 +635,72 @@ def start_3play_transcription_process(self, encoded_file):
                 self.video_query.studio_id,
             )
 
+    def _CIELO24_UPLOAD(self):
+        """
+        Note: This is part of the old flow which was deprecated when transcript phase 1 was reased.
+        """
+        # TODO: this must be removed once existing 3rd party credentials are migrated according
```

Comment:
@Qubad786 Would there be any issue now we are reverting this change?

Reply:
nope, but we have to make the credentials are migrated.